helm/prometheus-rules/templates/platform/atlas/recording-rules/signals.observability-platform.rules.yml

```
@@ -0,0 +1,119 @@
+          expr: |
+            sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, source_system, creation_mode, tenant) (
+              label_replace(
+                label_replace(
+                  label_replace(
+                    label_replace(
+                      label_replace(
+                        sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, user, rule_group) (
+                          loki_prometheus_rule_group_rules{cluster_type="management_cluster", rule_group=~".+"}
+                        ),
+                        "cluster_id", "$1", "rule_group", "/var/loki/rules-temp/[^/]+/[^-]+-[^-]+-(([a-z0-9]+(-[a-z0-9]+)*)-[a-z0-9]+((-[a-z0-9]+)*))\\.[^;]+;.*"
+                      ),
+                      "source_system", "loki", "", ""
+                    ),
+                    "creation_mode", "clickops", "", ""
+                  ),
+                  "creation_mode", "gitops", "rule_group", "/var/loki/rules-temp/[^/]+/[^-]+-[^-]+-[^-]+-[^;]+;.*"
+                ),
+                "tenant", "$1", "user", "(.*)"
+              )
+            )
+
+        # Grafana Cloud Exporter rules with enriched labels
+        - record: giantswarm:observability:signals:rules
+          expr: |
+            sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, source_system, creation_mode, tenant) (
+              label_replace(
+                label_replace(
+                  label_replace(
+                    sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, rule_group) (
+                      prometheus_rule_group_rules{cluster_type="management_cluster", rule_group=~".+"}
+                    ),
+                    "source_system", "grafana-cloud-exporter", "", ""
+                  ),
+                  "creation_mode", "gitops", "rule_group", "/etc/prometheus/rules/prometheus-mimir-to-grafana-cloud-rulefiles-0/[^-]+-[^-]+-[^.]+\\.yaml;.*"
+                ),
+                "tenant", "giantswarm", "", ""
+              )
+            )
+
+        # Grafana rules with enriched labels
+        - record: giantswarm:observability:signals:rules
+          expr: |
+            sum by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, source_system, creation_mode, tenant, org) (
+              label_replace(
+                label_replace(
+                  label_replace(
+                    grafana_alerting_rule_group_rules{cluster_type="management_cluster", rule_group=~".+"},
+                    "source_system", "grafana", "", ""
+                  ),
+                  "creation_mode", "clickops", "", ""
+                ),
+                "tenant", "unknown", "", ""
+              )
+            )
+
+        # Base fired alerts from Mimir Alertmanager with tenant enrichment
+        - record: giantswarm:observability:signals:mimir_fired_alerts
```

```suggestion
        - record: giantswarm:observability:signals:fired_alerts
```
grafana and mimir series will be different because of the `source_system` label.
And this way you don't need the `grafana or mimir` recording rule in `grafana-cloud.rules.yml`.